src/views/templates/view_steps_password.jade

```diff
@@ -16,4 +16,10 @@ block region
 
 block controls
     button(class="next", form="password-form")= t('step password submit')
-    .progression
```

Comment:
Doesn't this remove the progression on the page ? I think we need the progression

Reply:
No since it's also here https://github.com/cozy/cozy-onboarding-v3/blob/master/src/views/templates/_base.jade#L29. And I think that could be removed from other steps as well.

Reply:
Let's merge it then.